source/contracts/reporting/ReputationToken.sol

```diff
@@ -45,8 +46,14 @@ contract ReputationToken is DelegationTarget, Typed, Initializable, StandardToke
     }
 
     function migrateIn(address _reporter, uint256 _attotokens) public afterInitialized returns (bool) {
-        require(ReputationToken(msg.sender) == universe.getParentUniverse().getReputationToken());
+        IUniverse _parentUniverse = universe.getParentUniverse();
+        require(ReputationToken(msg.sender) == _parentUniverse.getReputationToken());
         balances[_reporter] = balances[_reporter].add(_attotokens);
+        // Only count tokens migrated toward the available to be matched in other universes. The bonus should not be added
+        universe.increaseRepAvailableForExtraBondPayouts(_attotokens);
```

Comment:
Curious: Why did you choose to track `rep_available_to_mint` instead of `rep_minted`?  If it were inversed and you instead just tracked REP minted on other universes sourced from this one then you would only need to update the storage variable when someone mints REP in a destination universe (max 3 times per universe per fork) rather than every time someone migrates REP.  This is a minor gas cost optimization, so may not be worth changing, but I'm curious if there was a reason to go this direction?

Reply:
Note: inverting the above _may_ make addressing the 3-universe problem above a bit easier.

Reply:
The comment above this actually describes why. I didn't want the 5% bonus to count toward this so that we actually stay zero-sum and no weirdness with timing gets involved. Otherwise we could just track amount minted and totalSupply as you are pointing out.

Reply:
Would things be simplified if we were OK with counting the 5% towards mintable volume?

Reply:
I'm fine with letting the 5% bonus count towards mintable volume if you think its worth it for simplicity.